eventmachine.gemspec

```diff
@@ -12,7 +12,12 @@ Gem::Specification.new do |s|
   s.files = `git ls-files README.md CHANGELOG.md GNU LICENSE rakelib ext java lib docs`.split
   s.test_files = `git ls-files tests examples`.split
 
-  s.extensions = ["ext/extconf.rb", "ext/fastfilereader/extconf.rb"]
+  if RUBY_PLATFORM =~ /java/
+    s.files << "lib/rubyeventmachine.jar"
+    s.platform = 'java'
+  else
+    s.extensions = ["ext/extconf.rb", "ext/fastfilereader/extconf.rb"]
+  end
```

Comment:
Hmm... A good question. I'm not sure. I wasn't aware that there were any interactions between specs and rubygems.org. I thought it was all evaluated at ruby runtime. A quick google doesn't seem to bring up anything obvious.